packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OvirtCredentialsEdit.tsx

```diff
@@ -165,6 +232,24 @@ export const OvirtCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onC
           isDisabled={insecureSkipVerify}
         />
       </FormGroup>
+      {insecureSkipVerify ? (
+        ''
+      ) : (
+        <Hint>
+          <HintBody>
+            {cacertHelperTextMsgs.hint}
+            <Button
```

Comment:
please move the button inside the hint msg

Reply:
For supporting a href link, I can't add the link text into the text message, but rather just replace `Button` with `ExternalLink` and wrap both with the `Trans` component.
The text message split is still required, please see the fix. 
Any other idea?